### PR TITLE
CKEditor5 - editor.getData() not taking into account the image format

### PR DESCRIPTION
**Pull request by xjiang-at-wiris**
## Description

> Please include a summary of the changes and the related issue. Please also include relevant motivation and context. List any dependencies that are required for this change. Include, if necessary, a description of the proposed solution and the reasoning behind it.

- **Related Kanbanize Card:** [Link to KB-50469](https://wiris.kanbanize.com/ctrl_board/2/cards/50469/details/)

## Type of Change

> Please delete options that are not relevant.

- [ ] Feature (non-breaking change which adds functionality)
- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] Chore (non-breaking change that doesn't add any functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update (changes to documentation only)
- [ ] Refactoring (non-breaking change that improves the code structure)

## Checklist

- [X] My code follows the style guidelines of this project ( Run `yarn lint` to check)
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation (if applicable)
- [ ] My changes generate no new warnings or errors
- [ ] I have added tests that prove my fix is effective or that my feature works (if applicable)
- [ ] New and existing unit tests pass locally with my changes

## How should be tested? (Manual or Automated Tests)

> Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce,also list any relevant details for your test configuration.

- **Manual tests:** I have tested this changes manually.

### CODE REVIEW

File: packages/devkit/src/mathml.js [hunk 1]
```diff
@@ -292,6 +292,23 @@ export default class MathML {
     return mathml.replace(semanticsStartingTagRegex, "").replace(semanticsEndingTagRegex, "");
   }
```
Comment (usantos-at-wiris) [inline packages/devkit/src/mathml.js:294]: A simple vista, esto no hace lo mismo que la funcion de abajo ?

**Review by usantos-at-wiris — APPROVED** [review]

File: packages/ckeditor5/src/plugin.js [hunk 2]
```diff
@@ -414,40 +471,49 @@ export default class MathType extends Plugin {
...
-        const regexp = /<math(.*?)<\/math>/gm;
+        const regexp = /(<img\b[^>]*>)|(<math(.*?)<\/math>)/gm;
+        const formulas = [];
+        let formula;
 
         // Get all MathML formulas and store them in an array.
         // Using the conditional operator on data.main because the data parameter has different types depending on:
         //    editor.data.set can be used directly or by the source editing plugin.
         //    With the source editor plugin, data is an object with the key `main` which contains the source code string.
         //    When using the editor.data.set method, the data is a string with the content to be set to the editor.
```
Comment (carla-at-wiris) [inline packages/ckeditor5/src/plugin.js:500]: This comment is no longer true.
I'd update it to something like "Both `data.set` from the source plugin and console command are taken into account as the data received is an image containing the MathML."

**Review by carla-at-wiris — CHANGES_REQUESTED** [review]

File: packages/devkit/src/mathml.js (hunk 1 above)
Comment (carla-at-wiris) [reply packages/devkit/src/mathml.js:294]: Cambian los caracteres, de safe XML a XML normal. Yo me planteaba de unificar las 2 pero quizá es más sencillo tener 2 métodos bien separados que se entiendan correctamente cuando se usan

File: packages/ckeditor5/src/plugin.js (hunk 2 above)
Comment (xjiang-at-wiris) [reply packages/ckeditor5/src/plugin.js]: Updated! I also added comment about handling mathml

**Review by carla-at-wiris — APPROVED** [review]
Great value and all works great!

**Review by jgonzalez-at-wiris — APPROVED** [review]

**Review by carla-at-wiris — APPROVED** [review]

**Pull request merged**